tests/index.spec.ts

```diff
@@ -35,6 +35,31 @@ describe("addFormats options", () => {
     expect(validateTime("17:27:38Z")).toEqual(true)
     expect(validateTime("25:27:38Z")).toEqual(true)
     expect(validateTime("17:27")).toEqual(false)
+
+    const validateDatetime = ajv.compile({format: "date-time"})
+    expect(validateDatetime("2016-12-31T23:59:60Z")).toEqual(true)
+    expect(validateDatetime("2015-12-31t23:59:60Z")).toEqual(true)
+    expect(validateDatetime("2015-02-11t22:59:22Z")).toEqual(true)
+    expect(validateDatetime("2020-01-01T20:00:00.000Z")).toEqual(true)
+    expect(validateDatetime("2020-01-01T20:00:00.000Z")).toEqual(true)
+    expect(validateDatetime("2023-05-04T01:14:00+21:00")).toEqual(true)
+    expect(validateDatetime("2023-05-04T01:14:10+16:20")).toEqual(true)
+    expect(validateDatetime("2023-05-04T01:14:21+09:50")).toEqual(true)
+    expect(validateDatetime("2023-05-04T01:14:21-04:31")).toEqual(true)
+    expect(validateDatetime("2023-05-04T01:14:21-04:31")).toEqual(true)
```

Comment:
This is a duplicate to line 48.
```suggestion
    expect(validateDatetime("2023-05-04T01:14:21-23:59")).toEqual(true)